consensus/misc/kip71.go

```diff
@@ -51,8 +51,13 @@ func NextBlockBaseFee(parentHeader *types.Header, config *params.ChainConfig) *b
 	if parentGasUsed == gasTarget {
 		return new(big.Int).Set(parentHeader.BaseFee)
```

Comment:
@kjeom @bobpkr In the case of governance update, don't we need to consider that we need to return upper/lower bound base fee instead of parentHeader.BaseFee?